Circle_Sort/Circle_Sort.php

```diff
@@ -0,0 +1,85 @@
+<?php
+ 
+//In circle sort The elements of the array lying on the same circle diametrically opposite to each other are compared.
+
+function circleSortRec($a, $low, $high)
+{
+    $swapped = false;
+    
+    if($low == $high)
+    {
+        return false;
+    }
+
+    $lo = $low;
+    $hi = $high;
+
+    while($lo < $hi)
+    {
+
+        if($a[$lo] > $a[$hi])
+        {
+            $temp = $a[$lo];
+            $a[$lo] = $a[$hi];
+            $a[$hi] = $temp;
+            $swapped = true;
+        }
+        $lo++;
+        $hi--;
+    }
+      
+    if($lo == $hi)
+    if($a[$lo] > $a[$hi + 1])
+    {
+        $t = $a[$low];
+        $a[$low] = $a[$hi + 1];
+        $a[$hi + 1] = $t;
+        $swapped = true;
+    }
+
+    $mid = ($high - $low) / 2;
+    $firstHalf = circleSortRec($a , $low , $low + $mid);
+    $secondHalf = circleSortRec($a , $low + $mid + 1 , $high);
+    return $swapped || $firstHalf || $secondHalf;
+}
+
+function circleSort($a , $n)
+{
+
+    while(circleSortRec($a , 0 , $n - 1))
```

Comment:
Can't this be replaced by while(circleSortRec($a , 0 , $n - 1));

Reply:
@singh-shreya6  done the required changes.